### PR TITLE
comment and markdown content

### PR DESCRIPTION
**Pull request by gabriellelamarrlemee**
I added a few short paragraphs on commenting and markdown cells to the notebooks page.

### CODE REVIEW

File: docs/src/groupby.md [hunk 1]
```diff
@@ -61,7 +61,6 @@ accident_counts = accident_list.groupby("latimes_make_and_model").size().reset_i
 ```
 
 You can clean up the `0` column name assigned by pandas with the `rename` method. The `inplace` option, found on many pandas methods, will save the change to your variable automatically.
-When you execute a cell, the code or text inside it will be processed and the output will be displayed below the cell. If the cell contains code, it will be executed by the kernel (the Jupyter notebook's underlying programming language, such as Python) and any output generated by the code will be displayed.
```
Comment (palewire) [inline docs/src/groupby.md:64]: Good cut. This does seem like something we covered earlier in the class.

File: docs/src/notebook.md [hunk 2]
```diff
@@ -85,4 +92,19 @@ To remove a cell, you can select the cell and press the "dd" key. Alternatively,
 Cells can contain variables, functions or imports. If you’ve never written code before and are unfamiliar with those terms, we recommend [“An Informal Introduction to Python”](https://docs.python.org/3/tutorial/introduction.html) and subsequent sections of python.org's official tutorial.
 ```
 
+Everything we have done so far has been in code cells, the default cell type. We can also make text cells, which are useful for giving our notebooks some structure and organization. In the menu, there is a dropdown arrow where you can change a cell from code to Markdown — text that is styled using Markdown syntax.
```
Comment (palewire) [inline docs/src/notebook.md]: Here's a couple quick tweaks, which aim to expand on your thoughts a little. Let me know what you think. 

```suggestion
Everything we have done so far has been in code cells, the default cell type. We can also make text cells, which are useful for giving our notebooks some structure and organization. You can do this by manipulating the pulldown menu in the toolbar directly above the notebook. By default the input is set to "Code." Click the dropdown arrow and change it to ["Markdown,"](https://en.wikipedia.org/wiki/Markdown) a markup language for formatting text similar to HTML.  
```

File: docs/src/notebook.md [hunk 3]
```diff
@@ -85,4 +92,19 @@ To remove a cell, you can select the cell and press the "dd" key. Alternatively,
 Cells can contain variables, functions or imports. If you’ve never written code before and are unfamiliar with those terms, we recommend [“An Informal Introduction to Python”](https://docs.python.org/3/tutorial/introduction.html) and subsequent sections of python.org's official tutorial.
 ```
 
+Everything we have done so far has been in code cells, the default cell type. We can also make text cells, which are useful for giving our notebooks some structure and organization. In the menu, there is a dropdown arrow where you can change a cell from code to Markdown — text that is styled using Markdown syntax. 
+
+These cells allow you to create headers, write descriptions, add links, and more to add context to your code. ["The Ultimate Markdown Guide"](https://medium.com/analytics-vidhya/the-ultimate-markdown-guide-for-jupyter-notebook-d5e5abf728fd) is a reference on all of the styling you can include but for now, let's try adding a heading and a bulleted list.
```
Comment (palewire) [inline docs/src/notebook.md]: A couple more little tweaks.

```suggestion
These cells allow you to create headers, write descriptions, add links and more to add context to your code. ["The Ultimate Markdown Guide"](https://medium.com/analytics-vidhya/the-ultimate-markdown-guide-for-jupyter-notebook-d5e5abf728fd) is a reference on all of the styling that you can draw from. For now, let's try adding a heading and a bulleted list.
```

**Review by palewire — COMMENTED** [review]
This is a great addition. Thank you for doing it!

File: docs/src/groupby.md (hunk 1 above)
Comment (gabriellelamarrlemee) [reply docs/src/groupby.md:64]: Yep. It's in the notebook section. I flagged it to move it over and then noticed it was already there.

File: docs/src/notebook.md (hunk 2 above)
Comment (gabriellelamarrlemee) [reply docs/src/notebook.md]: Much better thank you.

**Comment by palewire** [comment]
Alright. Going out. Thanks again.

**Pull request merged**